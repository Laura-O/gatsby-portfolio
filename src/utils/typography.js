import Typography from 'typography';
import kirkhamTheme from 'typography-theme-kirkham';
kirkhamTheme.bodyFontFamily = ['Source Sans Pro', 'sans-serif']

const typography = new Typography(kirkhamTheme);
const { rhythm, scale } = typography;

export { rhythm, scale, typography as default };