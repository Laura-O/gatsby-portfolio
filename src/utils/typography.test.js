import typography from './typography';

test('exports', () => {
  expect(typography).toMatchSnapshot();
});
