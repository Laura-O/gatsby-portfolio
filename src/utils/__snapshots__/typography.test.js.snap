// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports 1`] = `
Object {
  "*": Object {
    "boxSizing": "inherit",
  },
  "*:after": Object {
    "boxSizing": "inherit",
  },
  "*:before": Object {
    "boxSizing": "inherit",
  },
  "@media only screen and (max-width:480px)": Object {
    "blockquote": Object {
      "marginLeft": "-1.0875rem",
      "marginRight": 0,
      "paddingLeft": "0.81562rem",
    },
    "html": Object {
      "fontSize": "100%",
    },
  },
  "a": Object {
    "color": "#ff5700",
    "textDecoration": "none",
  },
  "a:hover,a:active": Object {
    "color": "black",
  },
  "abbr": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
  },
  "abbr[title]": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
    "textDecoration": "none",
  },
  "acronym": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
  },
  "address": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "b": Object {
    "fontWeight": 700,
  },
  "blockquote": Object {
    "borderLeft": "0.27187rem solid hsla(0,0%,0%,0.9)",
    "color": "hsla(0,0%,0%,0.59)",
    "fontSize": "1.1487rem",
    "fontStyle": "italic",
    "lineHeight": "1.45rem",
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": "1.45rem",
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": "1.17812rem",
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "blockquote *:last-child": Object {
    "marginBottom": 0,
  },
  "blockquote > :last-child": Object {
    "marginBottom": 0,
  },
  "blockquote cite": Object {
    "color": "black",
    "fontSize": "1rem",
    "fontWeight": 400,
    "lineHeight": "1.45rem",
  },
  "blockquote cite:before": Object {
    "content": "\\"â€” \\"",
  },
  "body": Object {
    "MozFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "WebkitFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "color": "black",
    "fontFamily": "'Open Sans',sans-serif",
    "fontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "fontKerning": "normal",
    "fontWeight": 400,
    "msFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "wordWrap": "break-word",
  },
  "code": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.45rem",
  },
  "dd": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "dl": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "dt": Object {
    "fontWeight": 700,
  },
  "fieldset": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "figure": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "form": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "h1": Object {
    "color": "inherit",
    "fontFamily": "'Domine',sans-serif",
    "fontSize": "2rem",
    "fontWeight": 700,
    "lineHeight": 1.1,
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h1,h2,h3,h4,h5,h6": Object {
    "lineHeight": 1.1,
  },
  "h2": Object {
    "color": "inherit",
    "fontFamily": "'Domine',sans-serif",
    "fontSize": "1.51572rem",
    "fontWeight": 700,
    "lineHeight": 1.1,
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h3": Object {
    "color": "inherit",
    "fontFamily": "'Domine',sans-serif",
    "fontSize": "1.31951rem",
    "fontWeight": 700,
    "lineHeight": 1.1,
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h4": Object {
    "color": "inherit",
    "fontFamily": "'Domine',sans-serif",
    "fontSize": "1rem",
    "fontWeight": 700,
    "lineHeight": 1.1,
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h5": Object {
    "color": "inherit",
    "fontFamily": "'Domine',sans-serif",
    "fontSize": "0.87055rem",
    "fontWeight": 700,
    "lineHeight": 1.1,
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h6": Object {
    "color": "inherit",
    "fontFamily": "'Domine',sans-serif",
    "fontSize": "0.81225rem",
    "fontWeight": 700,
    "lineHeight": 1.1,
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "hgroup": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "hr": Object {
    "background": "hsla(0,0%,0%,0.2)",
    "border": "none",
    "height": "1px",
    "marginBottom": "calc(1.16rem - 1px)",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "html": Object {
    "boxSizing": "border-box",
    "font": "112.5%/1.45 'Open Sans',sans-serif",
    "overflowY": "scroll",
  },
  "iframe": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "img": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "maxWidth": "100%",
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "kbd": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.45rem",
  },
  "li": Object {
    "marginBottom": "calc(1.16rem / 2)",
  },
  "li *:last-child": Object {
    "marginBottom": 0,
  },
  "li > ol": Object {
    "marginBottom": "calc(1.16rem / 2)",
    "marginLeft": "1.45rem",
    "marginTop": "calc(1.16rem / 2)",
  },
  "li > p": Object {
    "marginBottom": "calc(1.16rem / 2)",
  },
  "li > ul": Object {
    "marginBottom": "calc(1.16rem / 2)",
    "marginLeft": "1.45rem",
    "marginTop": "calc(1.16rem / 2)",
  },
  "noscript": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ol": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "marginBottom": "1.16rem",
    "marginLeft": "1.45rem",
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ol li": Object {
    "paddingLeft": 0,
  },
  "p": Object {
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "p *:last-child": Object {
    "marginBottom": 0,
  },
  "pre": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.45rem",
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "samp": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.45rem",
  },
  "strong": Object {
    "fontWeight": 700,
  },
  "table": Object {
    "borderCollapse": "collapse",
    "fontSize": "1rem",
    "lineHeight": "1.45rem",
    "marginBottom": "1.16rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "width": "100%",
  },
  "td,th": Object {
    "MozFontFeatureSettings": "\\"tnum\\"",
    "WebkitFontFeatureSettings": "\\"tnum\\"",
    "borderBottom": "1px solid hsla(0,0%,0%,0.12)",
    "fontFeatureSettings": "\\"tnum\\"",
    "msFontFeatureSettings": "\\"tnum\\"",
    "paddingBottom": "calc(0.725rem - 1px)",
    "paddingLeft": "0.96667rem",
    "paddingRight": "0.96667rem",
    "paddingTop": "0.725rem",
    "textAlign": "left",
  },
  "th": Object {
    "fontWeight": 700,
  },
  "th:first-child,td:first-child": Object {
    "paddingLeft": 0,
  },
  "th:last-child,td:last-child": Object {
    "paddingRight": 0,
  },
  "thead": Object {
    "textAlign": "left",
  },
  "ul": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "marginBottom": "1.16rem",
    "marginLeft": "1.45rem",
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ul li": Object {
    "paddingLeft": 0,
  },
}
`;
